share/spice/launchpad_project/launchpad_project.js

```diff
@@ -0,0 +1,55 @@
+(function (env) {
+    "use strict";        
+    env.ddg_spice_launchpad_project = function(api_result){
+
+        if (!api_result || !api_result.entries || api_result.error || api_result.total_size === 0) {
+            return Spice.failed('launchpad_project');
+        }
+        
+        //display "Launchpad Porject" if there is only one project returned
+        if (api_result.total_size === 1) {
```

Comment:
We actually *just* made a function for this, which will be documented soon:

Here's an example:
```js
var amt = 1;
DDG.pluralize(amt, "leaf", "leaves"); //"leaf"

var amt = 3;
DDG.pluralize(amt, "leaf", "leaves") //"leaves"

var amt = 5;
DDG.pluralize(amt, "leaf") // "leafs" (when no plural form given, appends an "s" to singular form)
```